pages/security/human-security/message-hygiene/de.md

```diff
@@ -8,7 +8,7 @@ Der Hauptgrund, dass Emails die Hauptangriffsmöglichkeit einer Organisation sin
 
 Zur Wiederholung: Die "Von"-Adresse kann von allen beliebig manipuliert werden, sodass eine Email so aussieht, als käme sie von einer anderen Person.
 
-Weil die Herkunft einer Nachricht so schwer zu verifizieren ist, ist der Posteingang eine übliche Quelle für **[Phishing-Angriffe -> phishing]** und **[Schadsoftware->malware]**.
+Weil die Herkunft einer Nachricht so schwer zu verifizieren ist, ist der Posteingang eine übliche Quelle für **[Phishing-Angriffe -> phishing]** und **[Schadsoftware->viruses]**.
 
 * **Phishing** ist when someone sends an email claiming to be an entity they are not, and uses this deception to get information. They can be after social security numbers, bank information, passwords, or other sensitive information.
```

Comment:
Translation:
Von **Phishing** spricht man, wenn in einer Email eine falsche Absender-Identität vorgegaukelt und diese Täuschung benutzt wird, um an persönliche Informationen des Empfängers zu gelangen. Das können Kontonummern, Zugangsdaten und Passwörter, Sozialversicherungsnummern und viele andere vertrauliche Informationen sein.
End of translation

BTW:" I do not understand **[Phishing-Angriffe -> phishing]** und **[Schadsoftware->malware]**." Those arehyperlinks which will work in the final Help environment?

Reply:
yes, they point to [/de/email/scams/phishing](https://riseup.net/de/email/scams/phishing) and [/de/email/scams/viruses](https://riseup.net/de/email/scams/viruses)